ui/src/dashboards/apis/index.ts

```diff
@@ -154,3 +36,47 @@ export const updateView = async (
 
   return viewWithIDs
 }
+
+export const cloneUtilFunc = async (dash: Dashboard, id: string) => {
```

Comment:
we might want to make a ticket to take a look at this function, it's pretty gnarly

Reply:
mayhaps indeed